content/documentation/pages/5-feature-guides/2-batch/6-composed-task.md

```diff
@@ -0,0 +1,643 @@
+
+### Basic Authentication
+
+In the example below we will launch a composed task where the user provides the username and password.
+
+1. Launch composed task as shown below by pressing the `play` button next to the composed task definition that needs to be launched:
+   ![Set User Access Token](images/SCDF-composed-task-user-security-launch.png)
+1. Set the `dataflow-server-username` and `dataflow-server-password` as follows in the `Parameters` text box:
+   ![Launch Task](images/SCDF-composed-task-user-security-basic-launch.png)
+1. Now press the `Launch the task` button to launch the composed task.
+
+### User Access Token
+
+In the example below we will launch a composed task where the `dataflow-server-use-user-access-token` is set to `true`.
+
+1. Launch composed task as shown below by pressing the `play` button next to the composed task definition that needs to be launched:
+   ![Set User Access Token](images/SCDF-composed-task-user-security-launch.png)
+1. Set the `dataflow-server-use-user-access-token` as follows in the `Arguments` text box:
+   ![Launch Task](images/SCDF-composed-task-user-security.png)
+1. Now press the `Launch the task` button to launch the composed task.
+
+### Using your own access token
+
+In the case the composed task needs to be launched with a specific access token then pass the token using the `dataflow-server-access-token` property.
+
+1. Launch composed task as shown below by pressing the `play` button next to the composed task definition that needs to be launched:
+   ![Set User Access Token](images/SCDF-composed-task-user-security-launch.png)
+1. Set the `dataflow-server-access-token` as follows in the `Parameters` text box:
+   ![Set User Access Token](images/SCDF-composed-task-user-security-launch-token.png)
+1. Now press the `Launch the task` button to launch the composed task.
+
+# Passing Properties
+
+Spring Cloud Data Flow allows users to pass both application and deployment properties to `Composed Task Runner` as well as its child tasks.
+
+## Passing Properties to child tasks
+
+There are 4 components that make up the property:
+
+- Property Type - This tells Spring Cloud Data Flow if the property is either a `deployment` or a `app` type.
+  - Deployment properties - are instructions to the deployer responsible for deploying the child task.
+  - App properties - are properties passed directly the child task.
+- Composed Task Definition Name
+- Child Task App Name - The label or the name of the application to which the property should be applied.
+- Property Key - The key of the property that is to be set.
+
+To set a property `myproperty` for a child task named `task-a` in a composed task named `my-composed-task` for the following dsl:
+
+```
+task-a && task-b
```

Comment:
we should also discuss the more straightforward way of doing `task-a --myproperty=myval`.  I think that is the more common case with overriding that application property being an exception.